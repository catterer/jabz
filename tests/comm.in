[
    { "u64# ignored"  : 1234605616436508552 }
]
