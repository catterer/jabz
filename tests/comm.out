00000000  08 00 00 00 88 77 66 55  44 33 22 11              |.....wfUD3".|
0000000c
